include/aws/checksums/private/crc_util.h

```diff
@@ -21,4 +22,32 @@
         return val;                                                                                                    \
     }
 
+/* helper function to reverse byte order on big-endian platforms*/
+static inline uint32_t aws_swap_bytes_if_needed_32(uint32_t x) {
```

Comment:
trivial: maybe I'm just not familiar with the literature, but to me "swap_bytes_if_needed" doesn't indicate which way they would swap, or what would be needed to make them swap
```suggestion
static inline uint32_t aws_swap_bytes_to_le_if_needed_32(uint32_t x) {
```

Reply:
renamed to aws_bswap64/32_if_be. 
swap bytes or byte swap (bswap is common intrinsic name) is a fairly typical name for reversing order of all bytes.